integration/disk_cleanup_deletes_inactive_directories_test.go

```diff
@@ -50,6 +50,7 @@ func TestDiskCleansupInactiveFilesets(t *testing.T) {
 	log.Debug("disk cleanup test")
 	require.NoError(t, testSetup.startServer())
 	log.Debug("server is now up")
+	testSetup.waitUntilServerIsUp()
```

Comment:
This is superfluous I believe no?  `startServer` waits by calling `waitUntilServerIsUp` already.